coordinator/src/models/errors/validation-errors.ts

```diff
@@ -13,12 +14,12 @@ export class MedicationRequestValueError implements ValidationError {
   severity = "error" as const
   expression: Array<string>
 
-  constructor(fieldName: string, uniqueFieldValues: Array<string>) {
+  constructor(fieldName: string, uniqueFieldValues: Array<unknown>) {
```

Comment:
Can we make this generic instead of Unknown?

Reply:
Yeah makes sense. This is one of the places that `any` would actually be appropriate